src/clarity-angular/button/_toggles.clarity.scss

```diff
@@ -36,7 +21,7 @@ $toggle-transition-animation: ease-in;
             display: inline-block;
             position: relative;
             cursor: pointer;
-            height: $toggle-height;
+            height: $clr-toggle-height;
```

Comment:
This variable was not named with the `clr-` prefix because it was local to the file and was used for calculations to compute other values based on the height of the toggle. What we planned on doing was to prefix global variables but not the local variables. But not since we have different files that changes. Just a comment. No action needed but you can let me know what you think about this.